app/uk/gov/hmrc/economiccrimelevyregistration/views/AmendReasonView.scala.html

```diff
@@ -39,12 +39,12 @@
         eclReferenceForStartLink = eclReference
     ) {
 
+        <h1 class="govuk-heading-l">@messages("amendReason.h1")</h1>
```

Comment:
Errors always have to go above everything. Move the h1 down

Reply:
Done.